src/main/resources/config/liquibase/changelog/20230519181800_changelog.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.1" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-4.18.xsd">
+    <changeSet author="kotzian" id="20230519181800">
+        <addColumn tableName="exercise">
```

Comment:
Use the `programming_exercise_details` table for this.
The exercise table should only be used if it is also used for something other than programming exercises.